src/azure-cli/azure/cli/command_modules/cosmosdb/tests/latest/test_cosmosdb_commands.py

```diff
@@ -1410,10 +1412,77 @@ def test_cosmosdb_managed_service_identity(self, resource_group, key_vault):
             'identity_principal_id': identity_principal_id
         })
         self.cmd('az keyvault set-policy -n {kv_name} -g {rg} --object-id {identity_principal_id} --key-permissions get unwrapKey wrapKey')
-
+        
+        # System assigned identity tests
+        cmk_account = self.cmd('az cosmosdb create -n {acc} -g {rg} --locations regionName={location} failoverPriority=0 --assign-identity [system] --default-identity FirstPartyIdentity').get_output_in_json()
         cmk_account = self.cmd('az cosmosdb update -n {acc} -g {rg} --default-identity SystemAssignedIdentity').get_output_in_json()
```

Comment:
I am probably missing it, can you point me to the create db tests that assigns and sets default to user.